docs/source/reference/numpysupported.rst

```diff
@@ -499,6 +499,7 @@ The following top-level functions are supported:
 * :func:`numpy.intersect1d` (only first 2 arguments, ar1 and ar2)
 * :func:`numpy.linspace` (only the 3-argument form)
 * :func:`numpy.logspace` (only the 3 first arguments)
+* :func:`numpy.nan_to_num` (only the 3 first arguments)
```

Comment:
In case you decide to not support complex values, update this message to say it only support integer and float values.

Reply:
I updated the PR to support complex numbers, so this statement should be okay now.